pc_lookup.c

```diff
@@ -31,7 +31,25 @@ static char *svnid = "$Id$";
 
 #ifdef ENABLE_BFD
 #ifndef CEXTRACT
-#include "bfd.h"
+  #ifndef PACKAGE
```

Comment:
Probably it is better to move this sequence out in a separate file to avoid duplication